app/va/va_profile/va_profile_client.py

```diff
@@ -48,6 +54,77 @@ def init_app(
         self.ssl_key_path = ssl_key_path
         self.statsd_client = statsd_client
 
+    def get_contact_info(self, va_profile_id: 'RecipientIdentifier') -> 'ContactInformation':
+        recipient_id = transform_to_fhir_format(va_profile_id)
+        oid = OIDS.get(va_profile_id.id_type)
+        url = f'{self.va_profile_url}profile/v3/{oid}/{recipient_id}'
+        data = {'bios': [{'bioPath': 'contactInformation'}]}
+
+        try:
+            response = requests.post(url, json=data, cert=(self.ssl_cert_path, self.ssl_key_path), timeout=(3.05, 1))
+            # FOR DEBUGGING PURPOSES ONLY! DO NOT MERGE!!
+            current_app.logger.info('***************************')
+            current_app.logger.info(f'Retrieved Response from V3 Endpoint: {response.text}')
+            current_app.logger.info('***************************')
+            response.raise_for_status()
+        except Exception as e:
```

Comment:
I updated this code:
```
        except Exception:
            current_app.logger.info('***************************')
            current_app.logger.exception('Error retrieiving contact information:')
            current_app.logger.info('***************************')
```
... and the github action failed when deploying to dev.  reverted the change to sanity check